jitter/extract.nim

```diff
@@ -0,0 +1,43 @@
+import std/[strformat, strutils, osproc, os]
+
+import zippy/tarballs
+import zippy/ziparchives
+
+import log
+
+let baseDir = getHomeDir() & ".jitter/"
+let nerveDir = baseDir / "nerve"
+let binDir = baseDir / "bin"
```

Comment:
add `bin/`, otherwise installs symlinks to parent directory with prefix `bin` (error)

Reply:
might need to add same thing to `nerveDir`

Reply:
That cannot happen because the `/` procedure removes trailing slashes so it's essentially the same.

Reply:
That's not possible, the `/` proc removes trailing slashes.